# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../kannel/kannel-1.4.3-mysql-5.5-header.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix compiling when MySQL >= 5.5

Starting with MySQL 5.5 the mysql.h file is installed into $includedir and
not into $includedir/mysql

--- gateway-1.4.3/gw/dlr_mysql.c.orig	2011-03-16 16:47:10.308072241 +0100
+++ gateway-1.4.3/gw/dlr_mysql.c	2011-03-16 16:48:21.180072241 +0100
@@ -71,7 +71,7 @@
 
 
 #ifdef HAVE_MYSQL
-#include <mysql/mysql.h>
+#include <mysql.h>
 
 /*
  * Our connection pool to mysql.
